src/test/java/com/catcher/core/service/ScheduleServiceTest.java

```diff
@@ -0,0 +1,323 @@
+
+    @Autowired
+    UserItemRepository userItemRepository;
+
+    @Autowired
+    TagRepository tagRepository;
+
+    @Autowired
+    LocationPort locationPort;
+
+    @PersistenceContext
+    EntityManager em;
+
+    @Autowired
+    private ScheduleService scheduleService;
+
+    @DisplayName("SUCCESS: 임시 저장된 일정 조회")
+    @Test
+    void get_draft_schedule() {
+        // Given
+        User user = createUser(createRandomUUID(), createRandomUUID(), createRandomUUID(), createRandomUUID());
+        userRepository.save(user);
+
+        Location location = locationPort.findByAreaCode("1100000000")
+                .orElseThrow();
+
+        Schedule draftSchedule = createSchedule(user, location, ScheduleStatus.DRAFT);
+        Schedule savedSchedule = createSchedule(user, location, ScheduleStatus.NORMAL);
+
+        scheduleRepository.save(draftSchedule);
+        scheduleRepository.save(savedSchedule);
+        flushAndClearPersistence();
+
+        // When
+        DraftScheduleResponse result = scheduleService.getDraftSchedule(user);
+
+        // Then
+        assertEquals(1, result.getSchedules().size());
+        assertEquals(draftSchedule.getTitle(), result.getSchedules().get(0).getTitle());
+    }
+
+
+
+    @DisplayName("SUCCESS: 임시 저장된 일정 조회 시, 임시 저장된 일정이 없을 경우")
+    @Test
+    void get_none_draft_schedule() {
+        // Given
+        User user = createUser(createRandomUUID(), createRandomUUID(), createRandomUUID(), createRandomUUID());
+        userRepository.save(user);
+        flushAndClearPersistence();
+
+        // When
+        DraftScheduleResponse result = scheduleService.getDraftSchedule(user);
+
+        // Then
+        assertEquals(0, result.getSchedules().size());
+    }
+
+    @DisplayName("SUCCESS: 일정 기본 정보 저장")
+    @Test
+    void save_schedule_skeleton() {
+        // Given
+        User user = createUser(createRandomUUID(), createRandomUUID(), createRandomUUID(), createRandomUUID());
+        userRepository.save(user);
+
+        SaveScheduleSkeletonRequest request = SaveScheduleSkeletonRequest.builder()
+                .title("제목")
+                .thumbnail("image.png")
+                .location("서울특별시")
+                .startAt(ZonedDateTime.now(ZoneId.of("Asia/Seoul")))
+                .endAt(ZonedDateTime.now(ZoneId.of("Asia/Seoul")))
+                .build();
+
+        // When
+        SaveScheduleSkeletonResponse result = scheduleService.saveScheduleSkeleton(request, user);
+        flushAndClearPersistence();
+
+        // Then
+        assertThat(result).isNotNull();
+        assertThat(result.getId()).isNotNull();
+
+        List<Schedule> savedSchedules = scheduleRepository.findByUserAndStatus(user, ScheduleStatus.DRAFT);
+        assertEquals(1, savedSchedules.size());
+
+        Schedule savedSchedule = savedSchedules.get(0);
+        assertThat(savedSchedule.getTitle()).isEqualTo("제목");
+        assertThat(savedSchedule.getThumbnailUrl()).isEqualTo("image.png");
```

Comment:
이부분 request.getXXX()로 비교하는 것도 좋을거같아요

```suggestion
        assertThat(savedSchedule.getThumbnailUrl()).isEqualTo(request.getThumbnail());
```

Reply:
수정완료했습니다! 👍